met-web/src/components/engagement/form/EngagementWidgets/Documents/CreateFolderForm.tsx

```diff
@@ -39,17 +39,18 @@ const CreateFolderForm = () => {
     };
 
     const handleCreateFolder = async () => {
-        if (!widget || validate()) {
```

Comment:
Let's change the .some in line 29 to every and in the condition we can do if (!widget || !validate())